backend/index.js

```diff
@@ -193,8 +229,63 @@ app.get("/api/categories", async (req, res) => {
   }
 });
 
-// Process JSON file
-processJsonFile();
+//Interact route
```

Comment:
appreciate the comment! maybe a bit more clarity would be even more helpful, something like "Interact route - used for various interactions between users and businesses"